test/BUILD

```diff
@@ -17,3 +17,19 @@ pytest_test(
         requirement("jsonschema")
     ],
 )
+
+pytest_test(
+    name = "test_overall_reports",
+    srcs = ["test_overall_reports.py"],
+    deps = [
+        "//:p2p"
```

Comment:
```suggestion
        "//reports",
        requirement("pandas"),
```

Reply:
I tried before pushing without `requeriment("pandas")" and it didn't work. I will try as you said.